pkg/relayer/proxy/proxy_test.go

```diff
@@ -462,6 +463,49 @@ func TestRelayerProxy_Relays(t *testing.T) {
 			expectedErrCode: 0,
 			expectedErrMsg:  "",
 		},
+		{
+			desc: "Successful late relay from session under grace period",
+
+			relayerProxyBehavior: []func(*testproxy.TestBehavior){
+				// blockHeight is past the first session but within its grace period
+				testproxy.WithRelayerProxyDependenciesAndCurrentBlockHeight(
+					supplierKeyName,
+					blockHeight+sessionkeeper.SessionGracePeriod,
+				),
+				testproxy.WithRelayerProxiedServices(proxiedServices),
+				testproxy.WithDefaultSupplier(supplierKeyName, supplierEndpoints),
+				testproxy.WithDefaultApplication(appPrivateKey),
+				// Add 2 sessions with the first one being past but within the grace period
+				// and the second one being the current session
```

Comment:
Is this correct? 

```suggestion
				// Add 2 sessions, with the first one being the current session and the second one
				// being within the session grace period
```

Reply:
Will update the comment but its the opposite, in the sense that the first one is the oldest (first created)